cfgov/unprocessed/css/organisms/header.less

```diff
@@ -62,7 +61,11 @@
 */
 
 .o-header {
+    min-width: 320px;
```

Comment:
Note to self: remove https://github.com/cfpb/cfgov-refresh/pull/1399/files#diff-8263f8fac724e6961ecfffd5ed5c96e5R8